static/bidder-params/pangle.json

```diff
@@ -8,9 +8,27 @@
       "type": "string",
       "description": "Access Token",
       "pattern": ".+"
+    },
+    "appid": {
+      "type": "string",
+      "description": "App ID",
+      "pattern": "[0-9]+"
```

Comment:
Would this be the same effect as an integer type? Do you expect appid and placementid to have leading zeros or exceed an int32's length?

Reply:
Placementid could exceeds an int32's length, appid don't. they're both set to be of string type because of a historical problem on our side. We prefer to make this compatible with the existing codes.